styx-api-service/src/test/resources/reference.conf

```diff
@@ -0,0 +1,10 @@
+# maximum allowed running timeout (styx.stale-state-ttls.running is the default)
```

Comment:
Nit: let's name this file as `styx-api.conf` to align with `styx-scheduler.conf`.